src/IssueViz.Security/Taint/TaintList/ViewModels/TaintIssuesControlViewModel.cs

```diff
@@ -48,6 +49,8 @@ internal interface ITaintIssuesControlViewModel : INotifyPropertyChanged, IDispo
 
         ICollectionView IssuesView { get; }
 
+        ITaintIssueViewModel SelectedIssue { get; set; }
```

Comment:
This was in the xaml but there was no actual property in the view model to match it:
https://github.com/SonarSource/sonarlint-visualstudio/blob/branch-taint/src/IssueViz.Security/Taint/TaintList/TaintIssuesControl.xaml#L91